### PR TITLE
Updates for Aff

### PR DESCRIPTION
**Pull request by natefaubion**
I'm going around prepping PRs for Aff v4.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -16,14 +16,16 @@
...
     "purescript-strings": "^3.0.0",
     "purescript-prelude": "^3.0.0",
     "purescript-transformers": "^3.0.0",
     "purescript-monoid": "^3.0.0",
     "purescript-foldable-traversable": "^3.0.0",
-    "purescript-js-timers": "^3.0.0",
     "purescript-pipes": "^4.0.0"
+  },
+  "resolutions": {
+    "purescript-aff": "^4.0.0"
```
Comment (natefaubion) [inline bower.json]: This is because pipes requires Aff to define a `MonadAff` instance.

**Comment by justinwoo** [comment]
Could you update this PR sometime? Thanks!

**Comment by justinwoo** [comment]
In the meanwhile I pushed up https://github.com/justinwoo/purescript-spec/commit/3a0ac612af654ff74e9183fd1d46b8ef8b505f8f, but it needs downstream Pipes and IO updates.

**Comment by felixSchl** [comment]
@justinwoo pipes and io are both updated now (both on v5 now). Would you be able to update? @owickstrom which version should we release this under? Considering the major changes to aff, do we have to do a major bump as well?

**Comment by owickstrom** [comment]
@felixSchl I think this will warrant a new major.

**Comment by justinwoo** [comment]
Pushed an update with a rebase here: https://github.com/justinwoo/purescript-spec/commit/b558e0d613a39a81da8548c0129968ecae95722f

**Comment by felixSchl** [comment]
@justinwoo would you like to create a new PR?

**Comment by felixSchl** [comment]
Merged via #64. Thanks @natefaubion and @justinwoo

**Pull request closed without merging**